setuptools/tests/test_config_discovery.py

```diff
@@ -159,15 +254,24 @@ class TestNoConfig:
     EXAMPLES = {
         "pkg1": ["src/pkg1.py"],
         "pkg2": ["src/pkg2/__init__.py"],
-        "ns.nested.pkg3": ["src/ns/nested/pkg3/__init__.py"]
+        "pkg3": ["src/pkg3/__init__.py", "src/pkg3-stubs/__init__.py"],
+        "pkg4": ["pkg4/__init__.py", "pkg4-stubs/__init__.py"],
+        "ns.nested.pkg1": ["src/ns/nested/pkg1/__init__.py"],
+        "ns.nested.pkg2": ["ns/nested/pkg2/__init__.py"],
     }
 
     @pytest.mark.parametrize("example", EXAMPLES.keys())
     def test_discover_name(self, tmp_path, example):
         _populate_project_dir(tmp_path, self.EXAMPLES[example], {})
+        dist = _get_dist(tmp_path, {})
+        dist.get_name() == example
```

Comment:
assert ?

Reply:
Yeap, definitely.  Thanks for spotting that!

I will include the fix in a follow up PR I am currently preparing.